### PR TITLE
Fix Docs links

### PR DESCRIPTION
**Pull request by AndersonQ**
## What does this PR do?

Reverts the `Docs` link from `BEATNAME/modules.d/MODULE.yml.disabled` to use `master` instead of `main` introduced by #35226.
It also reverts https://github.com/elastic/beats/pull/35693 as it did not fix the problem.

## Why is it important?

The links are broken

## Checklist

- ~~[ ] My code follows the style guidelines of this project~~
- ~~[ ] I have commented my code, particularly in hard-to-understand areas~~
- [x] I have made corresponding changes to the documentation
- ~~[ ] I have made corresponding change to the default configuration files~~
- ~~[ ] I have added tests that prove my fix is effective or that my feature works~~
- ~~[ ] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.~~

## Related issues

- Relates #35226
- Closes https://github.com/elastic/ingest-dev/issues/2111

### CODE REVIEW

**Review by cmacknz — APPROVED** [review]

**Comment by cmacknz** [comment]
You are blocked by the code owner check from SEI. The lint check failing doesn't block merge.

> Waiting on code owner review from elastic/security-external-integrations

**Review by efd6 — APPROVED** [review]

**Pull request merged**
